src/Packages/java/io/PrintStream.php

```diff
@@ -11,8 +11,22 @@ class PrintStream
 {
     private $sequence = '';
 
-    public function println($arg)
+    public function println(...$arguments)
     {
+        $length = count($arguments);
+        if ($length > 1) {
+            throw new \InvalidArgumentException(
```

Comment:
I wanted you to make a new file into `\PHPJava\Exceptions` namespace.
PHPJava needs wrap any exceptions because root PHP namespace is disorderly.
But, it take do it another time.
(Sorry, I didn't think that root namespace)

Reply:
No problem. What do we name it?
(Is \PHPJava\Exceptions\InvalidArgumentException ok?)

Because this is a kind of logic exception (i.e. if this exception is thrown it means a bug in PHPJava), I think Java's exception should not be used here.

Reply:
Fixed in #200, but please tell me if you prefer another name.

Reply:
No problem, and great naming. Thanks!